libs/api/domains/mailchimp/src/lib/dto/mailchimpSubscribe.input.ts

```diff
@@ -0,0 +1,23 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsBoolean, IsEmail, IsOptional, IsString } from 'class-validator'
+
+@InputType()
+export class MailchimpSubscribeInput {
+  @Field()
+  @IsString()
+  signupID!: string
+
+  @Field()
+  @IsEmail()
+  email!: string
+
+  @Field({ nullable: true })
+  @IsOptional()
+  @IsString()
+  name?: string
+
+  @Field({ nullable: true })
+  @IsOptional()
+  @IsBoolean()
```

Comment:
Don't need IsBoolean + IsOptional + IsString for GraphQL inputs. Creates unnecessary noise for the reader.

Only valid validation is `IsEmail`. Other basic validation will be picked up by the GraphQL Schema.